src/main/resources/pdiresources/jobs/other-transformations.kjb

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
since this will manage the migration part of the process can we rename this to "migration-job.kjb"